pixel/head.html

```diff
@@ -1,11 +1,24 @@
-<script>
-  (function() {
-    var appId = "{{settings.gtmId}}";
-    if (!appId) {
-      console.error('Warning: No APP NAME ID is defined. Please configure it in the apps admin.');
+<script async="true" type="text/javascript" src="https://s.kk-resources.com/leadtag.js" ></script>
```

Comment:
Check if it's possible to add this snippet on the end of the <body> tag. If it's possible you can change the name of this file to `body.html` and this snippet will be included on the end of the <body> tag.

Reply:
The [docs](https://www.kelkoogroup.com/pt/suporte/suporte-a-comerciantes/manual-integration/#) say it is. I'll do it